pkg/services/x509/uzi_validator.go

```diff
@@ -0,0 +1,151 @@
+package x509
+
+import (
+	"crypto/x509"
+	"fmt"
+	"strings"
+
+	"github.com/lestrrat-go/jwx/jwa"
+
+	"github.com/nuts-foundation/nuts-auth/assets"
+	"github.com/nuts-foundation/nuts-auth/pkg/contract"
+	"github.com/nuts-foundation/nuts-auth/pkg/services"
+)
+
+// Uzi Attributes used to sign the message
+// See table 12 on page 62 of the Certification Practice Statement (CPS) UZI-register v10.x
+// https://zorgcsp.nl/Media/Default/documenten/2020-05-06_RK1%20CPS%20UZI-register%20V10.0.pdf
+var UziAttributes = []string{
+	"oidCa",
+	"version",
+	"uziNr",
+	"cardType",
+	"orgID",
+	"rollCode",
+	"agbCode",
+}
+
+type UziSignedToken struct {
+	jwtX509Token *JwtX509Token
+	contract     *contract.Contract
+}
+
+// UziValidator wraps a jwtX509Validator an can check Uzi signed JWTs.
```

Comment:
"wraps a jwtX509Validator" I think that's irrelevant for the comment since its just the inner workings